Week01/Problem02/cyu_02.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+"""This script is written by Chuanping Yu, on Jul 24, 2017,
```

Comment:
Typically, these docstrings are more useful for complete modules of code than short scripts like these, but I like that you're (literally) taking ownership of your work.  This sort of information, however (author, date, etc.), is better located in module *metadata*, which is data about the script itself.  This metadata can be created and stored by including these fields at the top of your scripts:
```
__author__ = "Chuanping Yu"
__date__ = "July 24, 2017"
```
Then, your docstring can be left for more descriptive content, like what exactly the program does:
```
"""This script computes the sum of even-valued Fibonacci numbers less than 4,000,000."""
```